test/lib/ufe/CMakeLists.txt

```diff
@@ -80,7 +80,6 @@ set(mayaPluginPath
 
 set(path 
     "${CMAKE_INSTALL_PREFIX}/lib"
-    "${MAYA_LOCATION}/bin"
```

Comment:
We don't need the path to bin directory. None of these tests needs it and ${MAYA_PY_EXECUTABLE} value is already populated with proper path.